sdk/search/Azure.Search.Documents/src/SearchClientOptions.cs

```diff
@@ -30,21 +30,27 @@ public enum ServiceVersion
             /// The 2020_06_30 version of the Azure Cognitive Search
             /// service.
             /// </summary>
-            V2020_06_30 = 1
+            V2020_06_30 = 1,
+
+            /// <summary>
+            /// The 2020_06_30_Preview version of the Azure Cognitive Search
+            /// service.
+            /// </summary>
+            V2020_06_30_Preview = 2,
             #pragma warning restore CA1707
         }
 
         /// <summary>
         /// The Latest service version supported by this client library.
         /// </summary>
-        internal const ServiceVersion LatestVersion = ServiceVersion.V2020_06_30;
+        internal const ServiceVersion LatestVersion = ServiceVersion.V2020_06_30_Preview;
 
         /// <summary>
         /// The service version to use when creating continuation tokens that
         /// can be passed between different client libraries.  Changing this
         /// value requires updating <see cref="Azure.Search.Documents.Models.SearchContinuationToken"/>.
         /// </summary>
-        internal const ServiceVersion ContinuationTokenVersion = ServiceVersion.V2020_06_30;
+        internal const ServiceVersion ContinuationTokenVersion = ServiceVersion.V2020_06_30_Preview;
```

Comment:
Revert this - it's a long discussion, but we shouldn't change this unless we change the algorithm.